src/Soil-Core/SoilTransaction.class.st

```diff
@@ -234,6 +234,8 @@ SoilTransaction >> commit [
 
 { #category : #actions }
 SoilTransaction >> continue [
+	"recordsToCommit ifEmpty: [ self crTrace: 'EMPTY recordsToCommit' ]."
```

Comment:
If you do a PR you should remove commented out stuff before, please

Reply:
Got it. Probably a bad habit. I should instead write more tests, and then trust the tests.

Reply:
It would still be good to remove it ;)

Reply:
I removed the commented out code. I thought you meant for future pull requests. I thought pull requests could not be changed, after submission, but it makes sense that they're actually easy to update. (My usage of github has been limited)

Reply:
A pull request refers to a branch that you can change and github will redo the build. It would be hard otherwise what to do if the build fails ;)